src/reducers/analyzeAreaSlice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const AnalyzeAreaSlice = createSlice({
+  name: 'analyzeArea',
+  initialState: {
+    isEmptyState: true,
+    isMore: {},
+    isItAGraph: true,
+    isSortASC: true,
+    sortBy: 'Resilience Hubs'
+  },
+  reducers: {
+    changeEmptyState: (state, action) => {
+      state.isEmptyState = !state.isEmptyState;
+    },
+    changeMore: (state, action) => {
+      state.isMore[action.payload] = !state.isMore[action.payload];
```

Comment:
isMore is initialized as an empty object (no keys or values), so for the first query of this, won't this cause an error?  There will be no such thing as !state.isMore[action.payload], so it seems like we should do a check for `action.payload in state.isMore` here and set it to `true` on the first call

Reply:
I think it's alright based on some little testing I just did. This won't get called unless the more button is actually pressed and then it is passed its key/value pair of Area#:True/False